lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -273,8 +274,8 @@ def test_linkedin_share_url(self):
             ('pfCertificationUrl', self.request.build_absolute_uri(test_url),),
         ])
         self.assertIn(
-            self.linkedin_url.format(params=urlencode(params)),
-            response.content
+            escape(self.linkedin_url.format(params=urlencode(params))),
+            unicode(response.content, 'utf-8')
```

Comment:
Why do we need to explicitly pass 'utf-8' to `unicode`?

Reply:
@amangano-edx  This test was throwing exception 
```
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 267: ordinal not in range(128)
```
The issue is js_escape filter is returning unicode string ,causing response.content byte string to be implicitly decoded to unicode string. As the response.content is having some unicode character the default codec 'ascii' is not able to decode it successfully. We have to explicitly decode via utf-8 before using assertIn. This is not needed in below test as it is not inserting any unicode. Simple ascii would get easily decoded implicitly via default codec.

Now the question is why the test was previously passing. My understanding is as there was no escaping, python was not forced to implicitly decode byte string.